framework/sendemail.py

```diff
@@ -62,19 +62,26 @@ def handle_outbound_mail_task():
   if settings.SEND_ALL_EMAIL_TO and to != settings.REVIEW_COMMENT_MAILING_LIST:
     to_user, to_domain = to.split('@')
     to = settings.SEND_ALL_EMAIL_TO % {'user': to_user, 'domain': to_domain}
-    # Don't add CC addresses for non-prod environments.
-    cc = None
+    if cc:
+      new_cc = []
+      for cc_addr in cc:
+        cc_user, cc_domain = cc_addr.split('@')
+        new_cc.append(
+            settings.CC_ALL_EMAIL_TO % {'user': cc_user, 'domain': cc_domain})
```

Comment:
Do we have a test for this?

Reply:
We do, but I explicitly added a comparison to the existing test to make sure this is constructed as expected 👍

Reply:
Thanks for doing it!